stdlib/public/core/StringBridge.swift

```diff
@@ -476,9 +505,9 @@ internal func _bridgeCocoaString(_ cocoaString: _CocoaString) -> _StringGuts {
   case .constantTagged:
     let taggedContents = getConstantTaggedCocoaContents(cocoaString)!
     return _StringGuts(
-      cocoa: cocoaString,
+      cocoa: taggedContents.untaggedCocoa,
       providesFastUTF8: false, //TODO: if contentsPtr is UTF8 compatible, use it
-      isASCII: taggedContents.asciiContentsPointer != nil,
+      isASCII: true,
       length: taggedContents.utf16Length
```

Comment:
Is this what we want, or do we want to bring it in as an immortal string? Have you tested that ref counting and message sending Swift-side works on that untagged object?

Reply:
I think we want to, but I'm trying to be conservative about the changes initially both to unblock people and to reduce risk. This limits the changes to just the bridging paths.

Reply:
(and yes, that's been tested, it's an ordinary NSCFConstantString)

Reply:
They'd also have be null-terminated.